packages/places/__tests__/dbWritter.test.js

```diff
@@ -1,3 +1,8 @@
+/*
+Here we are testing the dbWritter method with an entity that in this case
+is the Places entity
+*/
+
 const { postPlace } = require('../src/index');
 
 const newPlaceInfo = {
```

Comment:
nitpick: You can use [fakerjs](https://github.com/faker-js/faker) to generate this data